### PR TITLE
Quite a bit of refinements

### PR DESCRIPTION
**Pull request by osx86-ijb**
A fair and somewhat large amount of word refinement, and further elaboration. Quite a bit of fixes included as well.

### CODE REVIEW

**Comment by osx86-ijb** [comment]
Continued refinement of words, as well as added disclaimer for extra good measure. ;)

File: README.md [hunk 1]
```diff
@@ -3,14 +3,14 @@
 
 ## MacOS install using UnRaid
 
-This guide is for the computer user who would like to run macOS inside of a VM on UnRAID in efforts of circumventing the need for multiple patches on AMD based hardware and motherboards, whilst enhancing the goals of providing the most native experience possible.
+This guide is for the AMD motherboard/cpu user who would like to run macOS from a VM inside of UnRAID Server OS, in efforts of circumventing the need for multiple patches on AMD based hardware and motherboards when booting bare metal from OpenCore.
```
Comment (Pavo-IM) [inline README.md:6]: This guide will work for both Intel and AMD systems. The XML settings and OpenCore config settings will work for both system types.

File: README.md [hunk 2]
```diff
@@ -33,15 +33,33 @@ This guide is for the computer user who would like to run macOS inside of a VM o
 
 ## FAQ
 
-#### 1) If I don't have an existing macOS installation to use to create an offline installer of macOS, yet am already booted into unRAID, what can I do to achieve such?
+#### 1) This sounds great! Where do I begin?
+[You can begin by making sure that you can adhere to the requirements outlined in this guide]
+
+#### 2) This is great, but UnRaid says that it costs money on the website?
+[That is correct, but as with all good things in life, there is a 30 day free trial, and the cheapest version costs a one time $60 USD payment. Gotta pay to play the game, right?]
```
Comment (Pavo-IM) [inline README.md]: You can use Unraid  or you can use any other Linux distro and install KVM and Qemu.

File: README.md [hunk 3]
```diff
@@ -51,15 +69,29 @@ This guide is for the computer user who would like to run macOS inside of a VM o
 - No hassle updating
 - macOS 12 Monterey Compatible
 
-  
-# Installation Procedurals
+## Requirements
+
+- An AMD based motherboard (AM4 or newer?)
```
Comment (Pavo-IM) [inline README.md:81]: This is not a requirement.

File: README.md [hunk 4]
```diff
@@ -51,15 +69,29 @@ This guide is for the computer user who would like to run macOS inside of a VM o
...
+- An AMD based central processing unit (Ryzen or newer?)
```
Comment (Pavo-IM) [inline README.md:82]: This is not a requirement.

**Review by Pavo-IM — COMMENTED** [review]
Please review the sections I highlighted.

**Pull request merged**